### PR TITLE
new blog on installing k8s delegate via minikube

### PR DESCRIPTION
**Pull request by krishi0408**

### CODE REVIEW

File: website/blog/2022-10-09-INSTALING_KUBERNETES_DELEGATE_VIA_MINIKUBE/installing_kubernetes_delegate.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+![Terminal](./1.png)
+
+- In windows you can check via running following commands in the command prompt and see if virtualization is available:
+
+    ```systeminfo```
+
+- Install kubectl (kubernetes command line )  
+  * Mac - ```brew install kubectl```
+  * Windows - For windows you can refer to the following docs for detailed instructions regarding installation:
+   [Kubectl-Windows](https://kubernetes.io/docs/tasks/tools/install-kubectl-windows/)
```
Comment (Ompragash) [inline website/blog/2022-10-09-INSTALING_KUBERNETES_DELEGATE_VIA_MINIKUBE/installing_kubernetes_delegate.md:28]: - Hypervisor technology (VirtualBox, VMWare, etc) is a mandate pre-requisite for Minikube and we have to choose the right one based on the platform we are on.
- Minikube will download the required `kubectl` as part of the installation and configures it.

With that said, all the above info along with the doc links to install, and configure the required hypervisor of our choice is provided in the Minikube documentation itself so it's safe to point to https://minikube.sigs.k8s.io/docs/start/#what-youll-need

File: website/blog/2022-10-09-INSTALING_KUBERNETES_DELEGATE_VIA_MINIKUBE/installing_kubernetes_delegate.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+- Installing minikube 
+   * Mac - ```brew install minikube``` 
+   * Windows -   Follow the following instructions for installation :
+    ![Wndows](./2.png)
+
+
+- Starting the Minikube
+
+   1. Mac and windows -  ```minikube start```
+   ![Mac and wind](3.png)
```
Comment (Ompragash) [inline website/blog/2022-10-09-INSTALING_KUBERNETES_DELEGATE_VIA_MINIKUBE/installing_kubernetes_delegate.md:57]: **Installation** section in the [Minikube Getting Started](https://minikube.sigs.k8s.io/docs/start/) documentation is well crafted and has steps for all Linux, Mac & Windows along with the architecture and installer type details and the user just has to choose the required details, get the commands and run them! So again, it's safe to point to [https://minikube.sigs.k8s.io/docs/start](https://minikube.sigs.k8s.io/docs/start/) and we can just focus on the deploy delegate part in this blog as rest looks perfect!

File: website/blog/2022-10-09-INSTALING_KUBERNETES_DELEGATE_VIA_MINIKUBE/installing_kubernetes_delegate.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+- You can also delete your delegate when no longer in use  
+
+## Note: Our Kubernetes Delegates are immutable , that is you can only create and delete the delegate but you can’t make any changes to them . 
+
+
+  
+
+
+
+
```
Comment (Ompragash) [inline website/blog/2022-10-09-INSTALING_KUBERNETES_DELEGATE_VIA_MINIKUBE/installing_kubernetes_delegate.md:105]:
```suggestion
```

**Review by Ompragash — CHANGES_REQUESTED** [review]
Overall looks great except for the Minikube steps as we can point to Minikube docs!

File: website/blog/2022-10-09-INSTALING_KUBERNETES_DELEGATE_VIA_MINIKUBE/installing_kubernetes_delegate.md (hunk 1 above)
Comment (krishi0408) [reply website/blog/2022-10-09-INSTALING_KUBERNETES_DELEGATE_VIA_MINIKUBE/installing_kubernetes_delegate.md]: ### Done thanks for the review !!

**Pull request merged**